src/pages/certifications/front-end-libraries/react/compose-react-components/index.md

```diff
@@ -1,10 +1,32 @@
 ---
-title: Compose React Components
+title:Compose React Components
```

Comment:
Hi @angelobarbano 

You need to add a space between `:` and the first letter of the title. It's not rendering correctly. Please see [this page](https://github.com/freeCodeCamp/guide/blob/6883a27a4e3c2fe04ed9c39b4b71ed5b69c3b5c8/src/pages/certifications/front-end-libraries/react/compose-react-components/index.md) to see how it's rendering.